client.lua

```diff
@@ -45,6 +45,28 @@ Citizen.CreateThread(function()
 		end
 end)
 
+Citizen.CreateThread(function()
+	while true do
+		if IsPlayerDead(PlayerId()) then
+			if high then 
+       			 StopGameplayCamShaking(true)
+				resetAnims()
+				DisabledRun = false
+				high = false
+				SetTransitionTimecycleModifier('default', 0.35)
+				--SetPedMoveRateOverride(source, 0)
+				SetRunSprintMultiplierForPlayer(source, 1.01)
+				SetEntityMaxHealth(GetPlayerPed(-1), 200)
+				TriggerEvent('chat:addMessage', {
+					color = { 255, 0, 0},
+					multiline = true,
+					args = {"WadL Drugs", "^2^*You have died under the influence"}
+				})
+			end
+		end
+		Citizen.Wait(3500)
+    end
+end)

```

Comment:
Fix the madness, You needed baseevents without this (but now its back to standalone)